.github/workflows/test-helm-book.yml

```diff
@@ -0,0 +1,238 @@
+name: Helm Docs Tutorials
+
+on:
+  push:
+    paths:
+      - "docs/book/src/cronjob-tutorial/testdata/project/**"
+      - "docs/book/src/getting-started/testdata/project/**"
+      - "docs/book/src/multiversion-tutorial/testdata/project/**"
+      - ".github/workflows/test-helm-book.yml"
+  pull_request:
+    paths:
+      - "docs/book/src/cronjob-tutorial/testdata/project/** "
+      - "docs/book/src/getting-started/testdata/project/**"
+      - "docs/book/src/multiversion-tutorial/testdata/project/**"
+      - ".github/workflows/test-helm-book.yml"
```

Comment:
Hi @akagami-harsh 

See that when we use the plugin we scaffold a test for it already. See: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v4-with-plugins/.github/workflows/test-chart.yml

So, could we not call the test that is scaffold for it already as we do for the e2e tests such as: https://github.com/kubernetes-sigs/kubebuilder/blob/master/.github/workflows/test-e2e-book.yml

Reply:
Note that for each case scenario we will need to uncomment the blocks https://github.com/kubernetes-sigs/kubebuilder/pull/4472/files#diff-3205e9fad2b66be3ac032e52423814f3505c57c9c39008ce966a2cde4be79229R49-R77 accordingly and to do so we can use the func https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugin/util/util.go as we do for other places in the hack/docs . Example: https://github.com/kubernetes-sigs/kubebuilder/blob/250d6775745e420f01a9ccfe80cb34adb5dad294/hack/docs/internal/cronjob-tutorial/generate_cronjob.go#L620-L623